.github/workflows/main.yml

```
@@ -27,7 +27,7 @@ jobs:
       - name: Check style
         run: |
           pipx run pycodestyle examples/*
-          pipx run ruff check --select=ASYNC,RUF006
+          pipx run ruff check
```

```suggestion
          pipx run ruff check  # default rules
          pipx run ruff check --select=ASYNC,RUF006  # Special rules for asyncio
```
This changes things.  Normally, `ASYNC` and `RUF` are ___off by default___, so this change removes those rules and adds the default rules.  https://docs.astral.sh/ruff/rules  Let's run both just to be safe.

---

Oh, I see. Thanks.